### DISCUSSION TITLE
Deploying Nextjs app on azure static web app.

### DISCUSSION THREAD

**Discussion by Puneet56** [nandorojo/solito#234] [General]
Hi, I am working on the project using the monorepo. 
Now, i have to deploy the nextjs app on azure. Here are the problems I am facing:

- Tried creating the static html export, does not works for dynamic pages.
- Tried normal export but azure cannot find build files, 
  - specfied the build directory to "/apps/next" does not work
  - specified distDir folder to 'build' as well as 'app' in next.config,js and "/apps/next/app" as build directory but still facing the issue.

Created a simple next js project and it works without any configuration along with dynamic paths, no static export
I do not want a static export.

![image](https://user-images.githubusercontent.com/26213419/201883736-97ffba31-e4c2-4384-9f53-3e726a662df5.png)

Any help will be appreciated. Thanks

**Comment by nandorojo** [comment]
why not use vercel?

**Reply by Puneet56** [reply]
The company uses Azure for all the other services and they do not want to move to any other service provider.
